packages/common/src/index.ts

```diff
@@ -462,4 +469,13 @@ export type ApiResponse =
     | ApiExploresResponse
     | ApiTableResponse
     | ApiStatusResponse
-    | ApiRefreshResponse
\ No newline at end of file
+    | ApiRefreshResponse
+    | ApiHealthResponse
+
+export type HealthState = {
+    healthy: boolean;
+    version: string;
+    latest: {
+        version?: string;
```

Comment:
Should this be: 
```
latest?: {version: string}
```

Reply:
I thought about it and went this way because is more flexible for the future. In case we want to have more properties that we might want to send to the FE. There is also the question, "if we can't get the latest version, will we get any other valuable information ?" which is a fair point. Thoughts ?

Reply:
Sounds good! Should this be `={}` then?

https://github.com/lightdash/lightdash/blob/f432ebfeba6066f658de8ce78c943ecadb873a2a/packages/backend/src/health.ts#L20